src/modules/user-auth/components/LoginForm/LoginForm.tsx

```diff
@@ -76,23 +79,32 @@ const LoginForm = () => {
       <ul aria-label="Sign-in options" tw="space-y-sm">
         <SignInExternalButton
           type="button"
-          icon={<FcGoogle css={signInExternalIconStyle} />}
+          icon={
+            <StyledIconWrapper>
+              <Icon icon={googleFill} tw="text-muted" />
```

Comment:
Let's use the icons in [this set](https://icon-sets.iconify.design/logos/) for Google, Facebook logo